azure-pipelines.yml

```diff
@@ -118,25 +118,26 @@ jobs:
   displayName: Build Software for Earl Grey toplevel design
   dependsOn: lint
   condition: and(succeeded(), eq(dependencies.lint.outputs['DetermineBuildType.onlyDocChanges'], '0'))
-  pool: ci-public
+  pool:
+    vmImage: ubuntu-18.04
   steps:
   - template: ci/install-package-dependencies.yml
   - bash: |
-      set -x
-      sudo util/get-toolchain.py \
-        --install-dir="$TOOLCHAIN_PATH" \
-        --release-version="$TOOLCHAIN_VERSION" \
-        --update
-    displayName: Install toolchain
-  - bash: |
-      . util/build_consts.sh
-      ./meson_init.sh -A
-      ninja -C "$OBJ_DIR" all
+      set -x -e
```

Comment:
Should this job be merged with the `bazel_test` job at the bottom of this file? It looks like that one's building all the sw and also running the unit tests.

Reply:
I'm ok with clean things up / optimizing after we remove meson completely if everyone else is too.

Reply:
I'll write a followup to handle this and address [#13005](https://github.com/lowRISC/opentitan/issues/13005)